### ISSUE TITLE
Confirmation button while enabling/disabling event through schedule page

### ISSUE THREAD

**Issue by tushar9953** [jhuckaby/Cronicle#647]
## Summary

Our teammate accidentally enabled a paused event. I guess, if we get a confirmation popup similar to the scheduler enable switch, it would be beneficial for all.

## Steps to reproduce the problem

Just click on enable checkbox on schedule page. There must be a confirmation required before toggling it.

## Your Setup

Cronicle is installed in a CentOS docker container.

### Operating system and version?

CentOS 7.9

### Node.js version?

v14.21.1

### Cronicle software version?

0.9.23

### Are you using a multi-server setup, or just a single server?
Yes

### Are you using the filesystem as back-end storage, or S3/Couchbase?
S3

### Can you reproduce the crash consistently?
Yes

**Comment by jhuckaby** [comment]
Hmmm, I personally don't like the confirmation dialog idea, because often times I need to enable/disable a bunch of events in a row, and I don't want to confirm a dialog box every time.

Would it be okay if I introduced an optional configuration property that simply hid the checkboxes on the schedule screen?  So the idea is, if you enable the feature, your users will have to click into each event to disable it, which requires extra steps, and will likely resolve your accidental click issue.

**Comment by tushar9953** [comment]
Hi @jhuckaby,

How about having a mixture of your idea and mine :p

If we can an optional configuration, if this property is enabled then only we will get a confirmation popup similar to scheduler enable switch. Would love to know other's suggestions too :)

**Comment by jhuckaby** [comment]
Forgive me, but adding a confirmation dialog for those checkboxes is just a can of worms I don't have time to open.  Cronicle v0.x is in maintenance mode, so I don't want to add any big new features (because I will have to re-do them in Cronicle v2, which has a completely different front-end UI).  

Those checkboxes are "real" OS-level checkboxes (form checkboxes) so they toggle off without me being able to intercept it and show a dialog.  So it gets into a can of worms because I have to track if the user dismisses the dialog (which they can do by clicking outside it, hitting ESC, or the cancel button), and so I'd have to add code to each of those pathways to RESET the checkbox to the other state if they cancel.  It just gets very hairy, and it's too much code to add to v0.x.

Whereas, just adding a config option to *hide* the checkboxes is 1000X easier, and can be done in 5 minutes.  So I have to go with that option for now, but I will address this properly in Cronicle v2, when it comes out later this year.

Sorry, but thank you very much for the suggestion.

**Comment by jhuckaby** [comment]
You might be able to get Mike to add the feature to his fork: https://github.com/cronicle-edge/cronicle-edge

**Comment by jhuckaby** [comment]
I added the ability to hide the checkboxes in [v0.9.35](https://github.com/jhuckaby/Cronicle/releases/tag/v0.9.35) if you are interested.  I know it's not what you asked for, but it's all I can do in this branch.

To hide the checkboxes, add this to your config.json file, inside the `client` section:

```js
"hide_schedule_checkboxes": true
```

Thanks, and sorry 😬

**Comment by tushar9953** [comment]
Thanks, @jhuckaby :)

**Comment by mikeTWC1984** [comment]
> You might be able to get Mike to add the dialog confirmation feature to his fork: https://github.com/cronicle-edge/cronicle-edge

Yeah, I actually added this feature already. I think it's useful if you share access to cronicle with a lot of other folks who are not too familiar with it (can disable something by accident). Although it's not that helpful for me, so I set this confirmation window to pop up only for the users who are not admins.  @tushar9953  I think you can port this to your current installation, by overwriting change_event_enabled function:

https://github.com/cronicle-edge/cronicle-edge/blob/main/htdocs/js/pages/Schedule.class.js#L997

**Comment by jhuckaby** [comment]
Awesome @mikeTWC1984 👏🏻 👏🏻 thank you
